docs/README.md

```diff
@@ -0,0 +1,31 @@
+# Local Chassis
+
+[Chassis](http://chassis.io/) is the bundled local development environment.
```

Comment:
Should we include how to use it? Like install deps for one? I know that's on the Getting Started, but could probably be embedded here too.